backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/impl/LanguageDAO.java

```diff
@@ -293,7 +295,7 @@ private int addTableTranslations(final String trlTableName, final String adLangu
 		//
 		// Insert Statement
 		final String trlAlias = "trl";
-		final int AD_User_ID = Env.getAD_User_ID(Env.getCtx());
+		final int AD_User_ID = CoalesceUtil.firstGreaterThanZero(Env.getAD_User_ID(Env.getCtx()), UserId.METASFRESH.getRepoId());
```

Comment:
I am a bit late to the party, but here you could use Env.getLoggedUserIdIfExists().orElse(UserId.METASFRESH).

@metas-rc @metas-ts  as a principle, the idea is to avoid using primitive `int`s as much as possible.
Ok, we need them when we load/save from/to model interfaces, but besides that, we shall avoid them by all means.